istio/hatch.toml

```diff
@@ -0,0 +1,18 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["27", "38"]
+version = ["1"]
```

Comment:
I'm not sure I understand this line. 

1. I think it's related to L9 in this file, right?
1. iiuc, this is the hatch equivalent of L6 in the tox file. Why are we switching from `1.13` to `"1"`?